notebooks/lesson08_group-project.ipynb

```diff
@@ -0,0 +1,1055 @@
+{
```

Comment:
<p>it occurred to me that we should add a subsection here with "Deliverables" and list the single jupyter notebook + video presentation + slides. I know we says this elsewhere, but it might help to collect it in one place</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>